src/cosp.F90

```diff
@@ -604,6 +606,7 @@ function COSP_SIMULATOR(cospIN,cospgridIN,cospOUT,start_idx,stop_idx,debug)
        Lmodis_subcolumn    = .true.
        Lcloudsat_column    = .true.
        Lcloudsat_subcolumn = .true.
+       Lcloudsat_modis_wr  = .true.
```

Comment:
Please add a comment here explaining that _wr means "warm rain".

Reply:
Yes, I added a comment line.